.gitignore

```diff
@@ -3,6 +3,7 @@
 
 # Anything pertaining to building the project
 build/
+.vscode/
```

Comment:
I don't like this here this should be in your personal gitignore. The way you do this is `git config --global core.excludesfile /path/to/your/personal/gitignore`

Reply:
Isn’t your suggestion excluding the folder from every project i’m working with? Some projects have ‘.vscode’ in their git. 

Additionally, isn’t this instantly give a better experience to anyone that downloads the repo and they don’t have to battle with their gitignore?